admin/resources/js/components/form/Select.tsx

```diff
@@ -20,29 +25,44 @@ const Select: React.FunctionComponent<SelectProps> = ({
   name,
   options,
   rules,
+  context,
   ...rest
 }) => {
   const {
     register,
     formState: { errors },
   } = useFormContext();
-  const error = errors[name]?.message;
+
+  const error = get(errors, name)?.message;
   return (
     <div>
-      <label htmlFor={id}>{label}</label>
-      <select
-        id={id}
-        {...register(name, rules)}
-        aria-invalid={error ? "true" : "false"}
-        {...rest}
+      <InputWrapper
+        inputId={id}
+        label={label}
+        required={!!rules?.required}
+        context={context}
+        error={error}
+        hideOptional
       >
-        {options.map(({ value, label: text }) => (
-          <option key={value} value={value}>
-            {text}
-          </option>
-        ))}
-      </select>
-      {error && <span role="alert">{error}</span>}
+        <select
+          data-h2-radius="b(s)"
+          data-h2-padding="b(top-bottom, xxs) b(left, xxs)"
+          data-h2-margin="b(bottom, xs)"
+          data-h2-font-size="b(normal)"
+          id={id}
+          style={{ width: "100%" }}
+          {...register(name, rules)}
+          aria-invalid={error ? "true" : "false"}
+          {...rest}
+        >
+          <option aria-label="Empty option" />
```

Comment:
I don't think the empty option should be a permanent part of the component, the parent may not want it, or want to control its text. They can include it in the options. We should add an optional `disabled?: boolean` prop to the options, giving the parent control of that as well.